Invalid package.json: JSONParseError: Unexpected end of JSON input while parsing empty string

```shell
» npx @tybalt/cli scaffold eleventy --name dougwade.io
Need to install the following packages:
@tybalt/cli@0.0.20
Ok to proceed? (y) y
npm WARN deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm WARN deprecated domexception@4.0.0: Use your platform's native DOMException instead
stdout:

Wrote to /Users/doug/code/dougwade.io/dougwade.io/package.json:

{
  "name": "dougwade.io",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}




installing dependencies; there may be errors
adding the following scripts to package.json
    test: run the unit tests
    build: build the static website
    serve: serve the local development website


node:internal/errors:932
  const err = new Error(message);
              ^

Error: Command failed: npm pkg set 'scripts.test'='jest'
npm ERR! code EJSONPARSE
npm ERR! JSON.parse Invalid package.json: JSONParseError: Unexpected end of JSON input while parsing empty string
npm ERR! JSON.parse Failed to parse JSON data.
npm ERR! JSON.parse Note: package.json must be actual JSON, not just JavaScript.

npm ERR! A complete log of this run can be found in: /Users/doug/.npm/_logs/2023-12-03T18_13_06_006Z-debug-0.log

    at ChildProcess.exithandler (node:child_process:422:12)
    at ChildProcess.emit (node:events:514:28)
    at maybeClose (node:internal/child_process:1105:16)
    at Socket.<anonymous> (node:internal/child_process:457:11)
    at Socket.emit (node:events:514:28)
    at Pipe.<anonymous> (node:net:337:12) {
  code: 1,
  killed: false,
  signal: null,
  cmd: "npm pkg set 'scripts.test'='jest'",
  stdout: '',
  stderr: 'npm ERR! code EJSONPARSE\n' +
    'npm ERR! JSON.parse Invalid package.json: JSONParseError: Unexpected end of JSON input while parsing empty string\n' +
    'npm ERR! JSON.parse Failed to parse JSON data.\n' +
    'npm ERR! JSON.parse Note: package.json must be actual JSON, not just JavaScript.\n' +
    '\n' +
    'npm ERR! A complete log of this run can be found in: /Users/doug/.npm/_logs/2023-12-03T18_13_06_006Z-debug-0.log\n'
}

Node.js v20.10.0
```

--- a/packages/cli/src/commands/scaffold.ts
+++ b/packages/cli/src/commands/scaffold.ts
@@ -18,10 +18,10 @@ import {
 } from '../templates/index.js';
 import { CommandContext, ScaffoldCommandOptions, ScaffoldContext, ScaffoldTarget } from '../types.js';
 
-const execAsync = async (command: string) => {
-    const promisifiedExec = util.promisify(exec);
+const promisifiedExec = util.promisify(exec);
 
-    const { stdout, stderr } = await promisifiedExec(command);
+const execAsync = async (command: string, opts = {}) => {
+    const { stdout, stderr } = await promisifiedExec(command, opts);
 
     if (stdout) {
         console.log(chalk.blue('stdout:\r\n'));
@@ -55,7 +55,7 @@ const writeScripts = async () => {
 
     await Promise.all(
         Object.entries(scripts).map(([scriptName, { script, description }]) => {
-            execAsync(`npm pkg set 'scripts.${scriptName}'='${script}'`);
+            execAsync(`npm pkg set 'scripts.${scriptName}'='${script}'`, { cwd: process.cwd() });
             console.log(`    ${scriptName}: ${description}`);
         }),
     );

